packages/sdk/package.json

```diff
@@ -63,17 +63,17 @@
   "dependencies": {
     "@azure/core-auth": "^1.3.0",
     "@azure/core-http": "^1.2.0",
-    "@azure/identity": "^1.3.0",
+    "@azure/identity": "^2.0.0",
     "@azure/msal-node": "~1.1.0",
     "@microsoft/microsoft-graph-client": "^2.2.1",
     "@microsoft/teams-js": "^1.9.0",
-    "axios": "^0.21.1",
-    "botbuilder": ">=4.9.4 <5.0.0",
-    "botbuilder-core": ">=4.9.4 <5.0.0",
-    "botbuilder-dialogs": ">=4.9.4 <5.0.0",
+    "axios": "^0.23.0",
+    "botbuilder": ">=4.15.0-rc0 <5.0.0",
+    "botbuilder-core": ">=4.15.0-rc0 <5.0.0",
+    "botbuilder-dialogs": ">=4.15.0-rc0 <5.0.0",
     "jwt-decode": "^3.1.2",
     "memory-cache": "^0.2.0",
-    "tedious": "^9.2.1",
+    "tedious": "^14.0.0",
```

Comment:
do you have fully test on this version?

Reply:
Not yet, and the latest version of tedious still have security issue.